### DISCUSSION TITLE
API to retrieve currently configured language?

### DISCUSSION THREAD

**Discussion by wita-sec** [OpenVoiceOS/ovos-buildroot#258] [Q&A] [answered]
Is there an API available to retrieve the currently configured language in OVOS?
The reason asking is that I wanted to generate texts by a skill that should match the language that has been configured in OVOS: see
ovos-config show -k /lang

**Comment by JarbasAl** [comment, marked as answer]
In a skill you can use self.lang , that will always reflect the current language for the received utterance in your intent

**Comment by wita-sec** [comment]
Perfect !
